_locales/zh-CN/messages.json

```diff
@@ -0,0 +1,157 @@
+{
+	"name":
+	{
+		"message": "搁置的标签页",
+		"description": "Extension name. Displayed in the manifest and pane header"
+	},
+	"description":
+	{
+		"message": "为Chromium浏览器提供旧版 Microsoft Edge 中\"搁置标签页\" 功能",
```

Comment:
Translation from Chinese: Provide the Chromium browser with the "hold tab" function _**in**_ the old version of Microsoft Edge
Correct translation: Provide the Chromium browser with the "hold tab" function _**from**_ the old version of Microsoft Edge

Reply:
Are you sure that it's right now? Seems like typo now (I'm not sure, sorry if I mistake)
![image](https://user-images.githubusercontent.com/28831743/87881550-ada15c00-ca02-11ea-9d6a-57be489f8ba6.png)
The first string is your original translation, the second one has been updated by you. And the third one I've tried to fix. And for the last one the translation seems more correct. Again, if you're sure that your version is correct I'll approve it as is

Reply:
Well I think both second and third translations are totally acceptable in Chinese since there is almost no difference in Chinese. You can use the one you prefer.

Reply:
Got it